submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/fire/ZipUtil.kt

```diff
@@ -22,6 +27,8 @@ object ZipUtil {
                     zs.closeEntry()
                 }
         }
+
+        checkZipIntegrity(sourceDir, zipFile)
```

Comment:
this is a generic zip util I think we should not include the double file check as part of the utility but rather do it in biostudies business code.